.env.defaults

```diff
@@ -14,6 +14,18 @@ KEEP_ALIVE_TIMEOUT_MS=
 # functioning correctly.
 CURVEBALL_ORIGIN="http://localhost:8531"
 
+# Allow CORS from any origin by running the seed script 002_cors_allowed_origin.js
+# ie. CORS_ALLOW_ORIGIN=["http://localhost:8531", "http://localhost:3000"]
+CORS_ALLOW_ORIGIN=
+
+# NOTE: Seed variables should only be set in development mode. 
+# Seed scripts can only run if tables don't already contain values.
+# Seed scripts WILL CLEAR EXISTING DATA.
+
+# Run user seed scripts only
+# Seed scripts WILL CLEAR EXISTING principals and principal_identities tables.
+SEED_USERS=
```

Comment:
I mentioned this earlier,  I think the seed scripts should only ever run by calling an explicit `make` target, not as part of the server script. Adding seed-data is a one-time choice a contributor might make, not something that's an 'always on flag' or server-configuration feature, which is what environment files are a bit more intended for.

You already did this with `make knex-seed`, so I really think that's enough.

Reply:
Ah I thought the server script command was necessary in order for this to work.

But accordingly if I'm changing the seed config in settings, it should run when the `knex seed:run ` is called I hope

Reply:
Putting this here doesn't make it "always on" though... it's so they know about what server env variables are at their disposal to set

Reply:
I meant that turning `SEED_USERS` off after the fact does not remove users. This makes it not a 'flag' but an action.

Reply:
Oooohhh!